config/prow/config.yaml

```diff
@@ -149,6 +149,21 @@ tide:
     - do-not-merge/release-note-label-needed
     - do-not-merge/work-in-progress
     - needs-rebase
+  - author: gardener-ci-robot
+    repos:
+    - gardener/ci-infra
+    labels: # gardener-ci-robot should only create autobump PR with this label
+    - skip-review
+    missingLabels:
+    - do-not-merge
```

Comment:
This label doesn't exist. I didn't copy it from test-infra, as it was marked as [deprecated](https://github.com/kubernetes/test-infra/blob/60549d3fa734ca0881514a2b764c4ac7f978a9f3/label_sync/labels.yaml#L166-L170).

Reply:
`do-not-merge` removed